feature/platform/transceiver/tests/zr_laser_bias_current_test/zr_laser_bias_current_test.go

```diff
@@ -143,26 +117,21 @@ func TestZRLaserBiasCurrentStateInterface_Flap(t *testing.T) {
 	i := d.GetOrCreateInterface(dp1.Name())
 	i.Enabled = ygot.Bool(false)
 	i.Type = oc.IETFInterfaces_InterfaceType_ethernetCsmacd
+	gnmi.Replace(t, dut1, gnmi.OC().Interface(dp1.Name()).Name().Config(), dp1.Name())
```

Comment:
can you please share more why this is needed as I think the next line would cover this?

Reply:
unintentional - removed the line now!